src/client/l10n/phoneNumbers.tsx

```diff
@@ -0,0 +1,19 @@
+import { MarketLabel } from './locales'
+
+type PhoneNumbers = Partial<Record<MarketLabel, PhoneNumberData>>
+
+export type PhoneNumberData = {
+  displayNumber: string
+  hrefNumber: string
+  opens: string
+  closes: string
+}
+
+export const phoneNumbers: PhoneNumbers = {
```

Comment:
Long term I think this should come from a CMS but okey for now :)

Reply:
Yes, Ellen and I had that discussion but came to the conclusion that this is good enough for now and then we'll see if it should be expanded to all markets etc.

Reply:
For sure! Totally agree